modules/nw-network-observability-operator.adoc

```diff
@@ -0,0 +1,90 @@
+/ Module included in the following assemblies:
```

Comment:
this line is showing in preview, is it supposed to?

Reply:
hmm. good catch. Thanks for noticing